kale/prepdata/image_transform.py

```diff
@@ -164,14 +164,32 @@ def mask_img_stack(images, mask):
     """Masking stacked images by a given mask
 
     Args:
-        images (array-like): input image data, shape (dim1, dim2, n_phases, n_subject)
+        images (array-like): input image data, shape (n_samples, n_phases, dim1, dim2)
         mask (array-like): mask, shape (dim1, dim2)
     Returns:
-        array-like tensor: masked images, shape (dim1, dim2, n_phases, n_subject)
+        array-like tensor: masked images, shape (n_samples, n_phases, dim1, dim2)
     """
-    n_phases, n_samples = images.shape[-2:]
+    n_samples, n_phases = images.shape[:2]
     for i in range(n_samples):
         for j in range(n_phases):
-            images[:, :, j, i] = np.multiply(images[:, :, j, i], mask)
+            images[i, j, ...] = np.multiply(images[i, j, ...], mask)
+
+    return images
+
+
+def normalize_img_stack(images):
+    """Normalization for stacked images
+
+    Args:
+        images (array-like tensor): Input data, shape (n_samples, n_phases, dim1, dim2).
+
+    Returns:
+        array-like: Normalized images, shape (n_samples, n_phases, dim1, dim2).
+    """
+    n_samples, n_phases = images.shape[:2]
+    for i in range(n_samples):
```

Comment:
No existing API to use for such a standard operation?

Reply:
There is one in OpenCV, but for a single image. So a function for stacked images is still required.